module/importer-ergonode-1/src/Infrastructure/Handler/Import/ImportDeletedCommandHandler.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\ImporterErgonode1\Infrastructure\Handler\Import;
+
+use Ergonode\Importer\Domain\Command\ImportDeletedCommand;
+use Ergonode\Importer\Infrastructure\Service\ImporterFileRemover;
+use Ergonode\ImporterErgonode1\Domain\Entity\ErgonodeZipSource;
+
+class ImportDeletedCommandHandler
+{
+    private ImporterFileRemover $importerFileRemover;
+
+    public function __construct(ImporterFileRemover $importerFileRemover)
+    {
+        $this->importerFileRemover = $importerFileRemover;
+    }
+
+    public function __invoke(ImportDeletedCommand $command): void
+    {
+        if (ErgonodeZipSource::TYPE === $command->getSourceType()) {
+            $this->importerFileRemover->remove($command->getFileName());
```

Comment:
Sam like in export Filename should be get from given Export not given data in command